### DISCUSSION TITLE
How to use Weather Icons?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#10167] [General - Components / Directives / etc] [answered]
I would like to integrate [Weather Icons](https://erikflowers.github.io/weather-icons/) into my application but despite trying all kinds of combinations they are not visible.

Weather Icons documentation states

> To use the Weather Icons, place the main CSS files in your CSS directory, and the font files in a "font" directory on the same folder level as the CSS director. Once you've done that, all you need to do to reference an icon in your HTML is type `<i class="wi wi-night-sleet"></i>`

Has anyone managed to successfully add them to Quasar?

**Comment by MilosPaunovic** [comment, marked as answer]
Check this out:

https://quasar.dev/options/quasar-icon-sets
https://quasar.dev/options/installing-icon-libraries

**Reply by wsw70** [reply]
Thank you! 

I see that it will be easiest to use the predefined sets of icons instead of forcing my own (some of the predefined ones have several weather icons and this will be "good enough" for my home dashboard).

**Reply by hawkeye64** [reply]
The bootstrap-icons has quite a few
